resources/rdf_dataset_exploration_queries/bq1_bq4_measures_and_topics.rq

```diff
@@ -0,0 +1,17 @@
+# Who has adopted what Covid19 measures and which issues they address?
+# What are the themes and topics applicable to measures?
+
+PREFIX sc: <http://publications.europa.eu/resource/ontology/sc#>
+PREFIX dct: <https://www.dublincore.org/specifications/dublin-core/dcmi-terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+SELECT DISTINCT ?measure ?title ?author ?author_label (GROUP_CONCAT (?topic_label; separator="; ") as ?topic_labels)
+WHERE {
+  ?measure a sc:Measure .
+  ?measure dct:author ?author .
+  ?author skos:prefLabel ?author_label .
+  ?measure dct:subject ?topic .
```

Comment:
I would add a distinction between dct:subject s. Why because we have PWDB classes, EU Vocab classes, EU timeline classes ... each shall be discriminated.